explainer-datamodel.md

```diff
@@ -119,66 +131,247 @@ FormattedText.format( [ "The quick ",
                       ], { style: "color: blue" } );
 ```
 
-A wide range of inline layout-related CSS is supported as style input to the `format` API.
+A wide range of inline layout-related CSS is supported as style input to the formatting functions.
 
-Styles specified in the second parameter apply to all text runs except where overridden on
-individual text objects (in the first parameter).
+Styles specified in the second parameter apply to all text runs except where the specific style
+property is also specified (overridden) on an individual text object (in the first parameter).
 
 ### Specifying constraints
 
-The final input to `format` are any constraints that should be applied to the formatted text.
-The primary constraints of interest to the API are the width and height of the containing block:
-how many CSS pixels are available for text layout.
+The inline-size constraint is the 3rd parameter for both formatting functions. `format` has
+an additional 4th parameter for the block-size constraint. Because the `format` function 
+produces all the lines of text at once, the block-size constraint allows the author to specify 
+a maximum bound for the container in both directions.
 
-The layout flow will follow any writing-mode direction specified in the meta object, so in
-`horizontal-lr` (e.g., English) a width constraint on the containing block will cause line 
-wrapping in the inline direction; overflow will occur in the vertical direction. For specified 
-writing-modes such as `vertical-rl` (e.g., Chinese) a width constraint on the containing block 
-will only affect where overflow occurs--the line will not wrap because it is assumed to have 
-infinite vertical space to layout. To cause wrapping with `vertical-rl` writing modes, specify
-a height constraint instead.
+To specify an inline-direction constraint for line wrapping with the `lines` iterator:
```

Comment:
Minor point, but I think this couple of paragraphs logically belongs in the next section (about `lines`), rather than this section (about `format`).

Reply:
Considering the flow for this section (on Constraints), I think it's OK to cover both lines/format definition of constraints in one place (rather than move it down into the more lines-specific area). So I don't think I'm going to adopt this suggestion.